openmp/libomptarget/src/device.cpp

```diff
@@ -33,6 +33,8 @@
 #include <string>
 #include <thread>
 
+[[gnu::constructor(101)]] void ctor() { printf("asdfasdf\n"); }
```

Comment:
What is this used for?

Reply:
Forgot I put that there to make sure that it was actually passing the flag still.